sympy/printing/str.py

```diff
@@ -981,19 +981,55 @@ def _print_AppliedBinaryRelation(self, expr):
 
 @print_function(StrPrinter)
 def sstr(expr, **settings):
-    """Returns the expression as a string.
-
-    For large expressions where speed is a concern, use the setting
-    order='none'. If abbrev=True setting is used then units are printed in
-    abbreviated form.
+    """
+    Returns the expression *expr* as a string.
+
+    Parameters
+    ==========
+    expr : :class:`.Expr`
+        Expression to be printed.
+    order : str or None, default: None
+        Any of the supported monomial orderings (currently 'lex',
+        'grlex', or 'grevlex'), 'old', and 'none'. This parameter does nothing
+        for :class:`.Mul` objects. Setting order to 'old' uses the
+        compatibility ordering for :class:`.Add` defined in :class:`.Printer`.
+        For very large expressions, set the *order* keyword to 'none' if
+        speed is a concern.
```

Comment:
Something I see a lot is users wanting expressions (especially unevaluated ones) to just print exactly as they are. To me that's the real reason someone might want to use `order='none'` so it is perhaps worth emphasising:
```python
In [4]: sstr(Add(y, x, evaluate=False))
Out[4]: 'x + y'

In [5]: sstr(Add(y, x, evaluate=False), order='none')
Out[5]: 'y + x'
```
I don't think I've ever seen anyone actually want the poly monomial orderings like `lex` and `grevlex`.